project/web/balancer.sls

```diff
@@ -170,6 +170,7 @@ install_letsencrypt:
   git.latest:
     - name: https://github.com/letsencrypt/letsencrypt/
     - target: {{ letsencrypt_dir }}
+    - force_checkout: True
```

Comment:
Do you think we might need `force_reset` as well? I'm not sure what part `git.latest` is failing on... the checkout or the fetch/merge.